### DISCUSSION TITLE
Vue Support

### DISCUSSION THREAD

**Discussion by syke99** [livebud/bud#39] [Ideas]
According to the demo video, bud is agnostic to front end frameworks, but currently only ships with Svelte and planned to ship with React in the future. Just curious if there are plans to ship with Vue as a friend of mine and I are currently working on a Go/Vue project

**Comment by j4v3l** [comment]
1+ .... I am also looking into working with Vue & Go

**Comment by matthewmueller** [comment]
Thanks for opening this. I know next to nothing about Vue, could you share what makes Vue special when compared with Svelte and React?

_(converted this to a discussion since it's not a bug)_

**Reply by stefanos82** [reply]
> I know next to nothing about Vue, could you share what makes Vue special when compared with Svelte and React?

Well, the author of Vue used to work for Google on Angular team and decided to write a framework that combined the best features (approaches) of Angular, Ember, and React.

**Reply by syke99** [reply]
> Thanks for opening this. I know next to nothing about Vue, could you share what makes Vue special when compared with Svelte and React? 

I’m a backend dev, so I’m not really too keen on the differences, myself. My front end team at work raves about how developer friendly it is. But yeah, I’ve heard that it is pretty much takes everything great from Angular and React and strips away the ugly parts. 

Also, I think opening up bud to shipping with Svelte, React, Angular, and Vue altogether would truly encompass the vast majority of web developers and improve adoption chances. I personally hate Angular from the little bit I've worked with it, but shipping with it would definitely intrigue the US Government to adopt it as they build all their front ends with Angular. And, at least in the Air Force realm, there are talks about considering moving most of their back ends from Java to Go that my friend (that I mentioned will be building a Vue/Go app with me; we worked together at a company contracted by the US Air Force) planted a couple of years ago. And I think bud could definitely help finish persuading them should it ship with Angular out of the box.

**Comment by chulinx** [comment]
I hope support vue too

**Reply by mikeenders77** [reply]
Svelte is easy enough for  a vue developer to pick up in a day.   I haven't seen many projects committing to the work of supporting  even two front end frameworks much less 4  (if angular and vue are to be included).  IMO  the eyes should now be on other pressing issues not on adding  other frameworks at this time.  Later fine but there are just so many hours in  day
